### ISSUE TITLE
Failing test: test_serialization.test_curried_exceptions

### ISSUE THREAD

**Issue by FRidh** [pytoolz/cytoolz#97]
The following test fails with cytoolz 0.8.2 and Python 2.7 and 3.5.

```
======================================================================
ERROR: test_serialization.test_curried_exceptions
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/nix/store/6hi80zli5wmih067bj51lcilgcx6jg62-python2.7-nose-1.3.7/lib/python2.7/site-packages/nose/case.py", line 197, in runTest
    self.test(*self.arg)
  File "/nix/store/k6phaf0pr611kj7yic8hmg6lvfacfbyv-python2.7-cytoolz-0.8.2/lib/python2.7/site-packages/cytoolz/tests/test_serialization.py", line 65, in test_curried_exceptions
    merge = pickle.loads(pickle.dumps(cytoolz.curried.exceptions.merge))
AttributeError: 'module' object has no attribute 'exceptions'
```

**Comment by eriknw** [comment]
Aha.  Thanks @FRidh.

This is the same issue as reported in pytoolz/toolz#357, and this was fixed--and tested--in `toolz` in pytoolz/toolz#358.  My bad in not being proactive; I should have done the same thing for `cytoolz`.

This error is annoying but benign.  Would you like a bug fix release?

**Comment by FRidh** [comment]
Hi @eriknw 

> This is the same issue as reported in pytoolz/toolz#357, and this was fixed--and tested--in toolz in pytoolz/toolz#358. My bad in not being proactive; I should have done the same thing for cytoolz.

No problem at all. We're all busy with things :)

> This error is annoying but benign. Would you like a bug fix release?

I've simply disabled this test in our build for now, so we don't necessarily need a release.

**Comment by eriknw** [comment]
`toolz` and `cytoolz` now ensure that bare test commands work, and version 0.9.0 was just released.  Closing.
